pkg/templates/templates.go

```diff
@@ -346,6 +346,11 @@ func (t *TemplateResolver) ResolveTemplate(tmplJSON []byte, context interface{})
 		"toBool":           toBool,
 	}
 
+	// Add all the functions from sprig we will support
+	for _, fname := range ExportedSprigFunctions {
+		funcMap[fname] = GetSprigFunc(fname)[0]
```

Comment:
If you're always grabbing the 0th index, could `GetSprigFunc` just return the function directly?